### PR TITLE
Namespace settings page

### PR DESCRIPTION
**Pull request by Alex-Tideman**
## What was changed
Add settings page to show namespace settings info. Page not included for cloud.

Screen Shot 2022-04-20 at 10 01 51 AM

## Why?
Let users see current namespace settings.

### CODE REVIEW

File: src/lib/utilities/format-date.ts [hunk 1]
```diff
@@ -149,3 +150,23 @@ export function getMilliseconds(date: ValidTime | undefined | null): number {
 
   return getSecondAsMilliseconds(parsedDate);
 }
+
+export function fromSecondsToDaysOrHours(seconds: string): string {
```
Comment (stevekinney) [inline src/lib/utilities/format-date.ts:154]: I think we can just used intervalToDuration.

File: src/routes/_header-responsive.svelte [hunk 2]
```diff
@@ -29,5 +30,10 @@
         Archival
       </NavigationLink>
     </IsCloudGuard>
+    <IsCloudGuard>
```
Comment (stevekinney) [inline src/routes/_header-responsive.svelte:33]: Can this just go in the same guard as above?

File: src/lib/utilities/format-date.ts (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/utilities/format-date.ts:153]: Updated to use intervalToDuration -> durationToString

File: src/routes/_header-responsive.svelte (hunk 2 above)
Comment (Alex-Tideman) [reply src/routes/_header-responsive.svelte]: Moved to same cloud check

File: src/routes/namespaces/[namespace]/settings/index@root.svelte [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<script context="module" lang="ts">
+  import type { Load } from '@sveltejs/kit';
+  import type { DescribeNamespaceResponse } from '$types';
+
+  export const load: Load = async function ({ params, url, stuff }) {
```
[the remark this answers is not in the document]
Comment (Alex-Tideman) [reply src/routes/namespaces/[namespace]/settings/index@root.svelte:5]: Yeah I tested with dev:cloud, menu doesn't show Settings and if you do go to settings page through url it loads the settings with all information as undefined.

**Review by stevekinney — APPROVED** [review]

**Pull request merged**